kubetest/main.go

```diff
@@ -289,6 +293,11 @@ func main() {
 		log.Fatalf("Flags validation failed. err: %v", err)
 	}
 
+	if o.version {
+		log.Printf("kubetest version: %s", gitTag)
```

Comment:
super nit: since this is the expected output of the command this should be written to stdout :)
(and a newline) e.g. for something like`kubetest --version 2>/dev/null`

maybe `fmt.Printf("kubetest version: %s\n", gitTag)` ?

Reply:
Ack

Reply:
might've missed `log` vs `fmt`

log defaults to stderr, fmt defaults to stdout.